pages/en/learn/getting-started/nodejs-with-typescript.md

```diff
@@ -117,6 +117,57 @@ TypeScript offers a whole lot of other great mechanisms like interfaces, classes
 
 Some of the other benefits of TypeScript that are worth mentioning are that it can be adopted progressively, it helps making code more readable and understandable and it allows developers to use modern language features while shipping code for older Node.js versions.
 
+## Running TypeScript code in Node.js
+
+Sadly node can do this `node example.ts`. But there are three solutions to this problem 🥳:
+
+### Compiling TypeScript to JavaScript
+
+If you want to run TypeScript code in Node.js, you need to compile it to JavaScript first. You can do this using the TypeScript compiler `tsc` as shown above.
+
+Here a small example:
+
+```bash
+npx tsc example.ts
+node example.js
+```
+
+### Running TypeScript code with ts-node
+
+You can use [ts-node](https://typestrong.org/ts-node/) to run TypeScript code directly in Node.js without the need to compile it first. But it's not typechecking your code. So we recommend to type check your code first with `tsc` and then run it with `ts-node` before shipping it.
+
+To use `ts-node`, you need to install it first:
+
+```bash
+npm i -D ts-node
+```
+
+Then you can run your TypeScript code like this:
+
+```bash
+npx ts-node example.ts
+```
+
+### Running TypeScript code with nodejs loader
+
+Since node v19.0.0 we had [custom loader](https://nodejs.org/docs/v20.11.1/api/cli.html#--importmodule). So you i'll need to download an loader such as `ts-node` or `tsx` or `nodejs-loaders`
+
+Firs you need to install the loader:
+
+```bash
+npm i -D ts-node
+```
+
+Then you can run your TypeScript code like this:
+
+```bash
+node --loader=ts-node/esm example.ts
+```
+
+## Conclusion
+
+TypeScript is a powerful tool that can help you write better code, catch bugs early and make your code more readable and maintainable. It's a great choice for Node.js projects and can be adopted progressively. TypeScript is widely used in the Node.js world and has a lot of great features that can help you write better code.
+
```

Comment:
this almost certainly was written by chat gpt or copilot. It repeats the phrasing of the following header in the last sentence, which is a common tell. suggest we remove the last section or clean this up

Reply:
that not GPT write x'D.
But yeah I had take basement on header. It's something we do in French to start from the beginning and finish.

Reply:
So you propose to remove it ?

Reply:
sorry for the accusation then 😄 

it reads as redundant to me

Reply:
> sorry for the accusation then 😄

No problem. I admit that I could have thought the same thing and there was still a deepl write on this paragraph.